lib/fog/voxel.rb

```diff
@@ -16,10 +16,10 @@ def self.create_signature(secret, options)
   end
 
   module Compute
-    autoload :Voxel, 'fog/compute/voxel'
+    autoload :Voxel, File.expand_path('../compute/voxel', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.